modules/random/include/hephaestus/random/random_object_creator.h

```diff
@@ -16,6 +16,7 @@
 
 #include "hephaestus/utils/concepts.h"
 #include "hephaestus/utils/exception.h"
+#include "hephaestus/types/bounds.h"
```

Comment:
@filippobrizzi there is an issue with our current module structure, which leads to cyclic dependencies. 

If we use `types::DummyType` to test other modules, then non of these modules must depend on types. If we add helper types like `Bounds<T>` to `types`, we cannot use them in most hephaestus modules. The random module is a good example.

Solutions: we either move DummyType into a separate module, like a test helper, or we try to formulate all helper types in terms of utility etc. The latter sounds more appealing.

`utility::???::Bounds<T>` <-- suggestions?

Reply:
makes sense, actually `Bounds` is not really a type, it is a helper/utils I would say.

Reply:
Any suggestion for a folder/namespace for `Bounds`? utils::types, utils::data, utils::ranges, ...

Reply:
I vote for `data`